src/providers/push/fcm/fcm.provider.ts

```diff
@@ -123,11 +124,10 @@ export class FcmProvider implements Provider<FcmNotification> {
           ...generalMessageObj,
           data: {...message.options.data},
         };
-        promises.push(
-          this.fcmService
-            .messaging()
-            .send(msgToTransfer, (message.options.dryRun = false)),
-        );
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const dryRun = message.options.dryRun ?? false;
+        const sendPromise = this.fcmService.messaging().send(msgToTransfer);
```

Comment:
The dryRun param was passed into the `.send` method earlier now it's removed, why?

Reply:
resolved